streaming-binlog/pom.xml

```diff
@@ -136,6 +137,23 @@
       <version>4.2.0</version>
       <scope>compile</scope>
     </dependency>
+    <!-- OpenMRS -->
+    <dependency>
+      <groupId>org.openmrs.api</groupId>
```

Comment:
Question: Why do we need the openmrs.api dependency here too? Is it because of the atomfeed-api? If it is needed we should think of a way to avoid it (please add a note/todo). This is also the case for the Atom Feed based implementation but that is more difficult to untangle. I was hoping that for the Debezium based approach we can completely avoid openmrs dependencies (I understand why the atomfeed-api dep is needed).

Reply:
Thanks, @bashir2 . atom-feed FeedConfigurationServicee extends from `org.openmrs.api.OpenmrsService` §Failure to include yields compilation error:
` cannot access org.openmrs.api.OpenmrsService
[ERROR]   class file for org.openmrs.api.OpenmrsService not found
`
I have add a note/TODO why this is needed.

Reply:
Okay, thanks Allan, I don't see your TODO though, did you forget to push your commit?

Reply:
Yet to push! Pushing in a few!